packages/docs-app/src/examples/datetime2-examples/dateInput2Example.tsx

```diff
@@ -151,27 +169,59 @@ export class DateInput2Example extends React.PureComponent<IExampleProps, DateIn
                 <H5>Appearance props</H5>
                 <Switch label="Disabled" checked={disabled} onChange={this.toggleDisabled} />
                 <Switch label="Fill" checked={fill} onChange={this.toggleFill} />
-                <Switch label="Reverse month and year menus" checked={reverse} onChange={this.toggleReverseMenus} />
-                <Switch
-                    label="Show time picker arrows"
-                    checked={showTimePickerArrows}
-                    disabled={timePrecision === undefined}
-                    onChange={this.toggleTimePickerArrows}
-                />
+                <PropCodeTooltip snippet={`reverseMonthAndYearMenus={${reverse.toString()}}`}>
+                    <Switch label="Reverse month and year menus" checked={reverse} onChange={this.toggleReverseMenus} />
+                </PropCodeTooltip>
+                <PropCodeTooltip
+                    snippet={`timePickerProps={{ showArrowButtons: ${showTimePickerArrows.toString()} }}`}
+                    disabled={!isTimePickerShown}
+                >
+                    <Switch
+                        label="Show time picker arrows"
+                        checked={showTimePickerArrows}
+                        disabled={!isTimePickerShown}
+                        onChange={this.toggleTimePickerArrows}
+                    />
+                </PropCodeTooltip>
+                <PropCodeTooltip
+                    snippet={`timePickerProps={{ useAmPm: ${useAmPm.toString()} }}`}
+                    disabled={!isTimePickerShown}
+                >
+                    <Switch
+                        label="Use AM/PM time"
+                        checked={useAmPm}
+                        disabled={!isTimePickerShown}
+                        onChange={this.toggleUseAmPm}
+                    />
+                </PropCodeTooltip>
                 <Switch label="Show right element" checked={showRightElement} onChange={this.toggleRightElement} />
                 <DateFnsFormatSelector format={format} onChange={this.handleFormatChange} />
 
-                <H5>Timezone props</H5>
-                <Switch
-                    label="Disable timezone select"
-                    checked={disableTimezoneSelect}
-                    onChange={this.toggleDisableTimezoneSelect}
-                />
-                <Switch
-                    label="Show timezone select"
-                    checked={showTimezoneSelect}
-                    onChange={this.toggleShowTimezoneSelect}
-                />
+                <H5 className={classNames({ [Classes.TEXT_DISABLED]: timePrecision === undefined })}>
```

Comment:
there's a CSS bug here -- filed for follow-up https://github.com/palantir/blueprint/issues/5448